DB/src/main/java/io/deephaven/db/tables/utils/TableTools.java

```diff
@@ -1661,4 +1661,26 @@ private static void processColumnForFingerprint(OrderedKeys ok, ColumnSource<?>
                 throw new UnsupportedOperationException();
         }
     }
+
+    public static String nullTypeAsString(final Class<?> dataType) {
```

Comment:
This is missing `byte`. Also, having seen what this is used for, I would delete it and use `io.deephaven.db.v2.select.NullSelectColumn`. Note that `NullSelectColumn` could use a constructor that takes component type, too.

Reply:
Demo or die.   Meaning, give me some sample code of how the function that uses this would have to look like for that to work, I have no idea how to use it.

Reply:
```
    private static Table pretransformTable(final Table table, final TableDefinition definition) {
        List<SelectColumn> updateViewColumnsTransform = new ArrayList<>();
        List<SelectColumn> viewColumnsTransform = new ArrayList<>();
        Table t = table;
        for (ColumnDefinition<?> column : definition.getColumns()) {
            final String colName = column.getName();
            if (t.hasColumns(colName)) {
                if (StringSet.class.isAssignableFrom(column.getDataType())) {
                    updateViewColumnsTransform.add(FormulaColumn.createFormulaColumn(colName, colName + ".values()"));
                }
                viewColumnsTransform.add(new SourceColumn(colName));
            } else {
                //noinspection unchecked
                viewColumnsTransform.add(new NullSelectColumn(column.getDataType(), colName));
            }
        }
        if (viewColumnsTransform.size() > 0) {
            t = t.view(viewColumnsTransform.toArray((SelectColumn.ZERO_LENGTH_SELECT_COLUMN_ARRAY)));
        }
        if (updateViewColumnsTransform.size() > 0) {
            t = t.updateView(updateViewColumnsTransform.toArray(SelectColumn.ZERO_LENGTH_SELECT_COLUMN_ARRAY));
        }
        return t;
    }
```

Reply:
Don't forget this part:
>Note that NullSelectColumn could use a constructor that takes component type, too.

Reply:
Done, PTAL and ensure the change for the NSC/NVCS ctor makes sense.